kafkamgmt/apiv1/api_client.go

```diff
@@ -0,0 +1,34 @@
+package kafkamgmt
+
+import (
+	"github.com/redhat-developer/app-services-sdk-go/internal"
+
+	apiv1 "github.com/redhat-developer/app-services-sdk-go/kafkamgmt/apiv1/client"
```

Comment:
Moved the generated OpenAPI code into a subfolder called "client" - I thought `generated` looked a little strange but I know that this is what the JS SDK uses. @wtrocki we should probably agree on a common one.

At least with the JS SDK the user does not see the `generated` folder as it is exported from index.ts, but in Go the generated folder needs to be impported directly.

Reply:
@craicoverflow I'm bad with names so can change it to client no problem in js-sdk anyway.